src/renderer/base/RenderSettings.cpp

```diff
@@ -192,22 +192,25 @@ std::pair<COLORREF, COLORREF> RenderSettings::GetAttributeColors(const TextAttri
     const auto swapFgAndBg = attr.IsReverseVideo() ^ GetRenderMode(Mode::ScreenReversed);
 
     // We want to nudge the foreground color to make it more perceivable only for the
-    // default color pairs within the color table
+    // default color pairs within the color table, and only if there's no additional text attributes
+
+    // The reason we don't want to nudge when there's additional attributes is because of certain
+    // interactions like "bright" + "reverse". We cannot nudge after reversing because we, as of now,

```

Comment:
i do think we should nudge bright foregrounds that aren't the default foreground, but... that might just be me? hrm.